containers/test-apps/waiter-init/bin/waiter-mesos-init

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Can we delete this file?

Reply:
Yes, this was supposed to be added in the WAITER_SANDBOX PR. Deleted.